lib/signaling/v2/peerconnection.js

```diff
@@ -904,6 +913,14 @@ class PeerConnectionV2 extends StateMachine {
     }).then(offer => {
       if (!isFirefox) {
         offer = workaroundIssue8329(offer);
+      } else {
+        // NOTE(mmalavalli): We work around Chromium bug 1106157 by disabling
+        // RTX in Firefox 79+. For more details about the bug, please go here:
+        // https://bugs.chromium.org/p/chromium/issues/detail?id=1106157
+        offer = new this._RTCSessionDescription({
```

Comment:
you want to apply this only for chrome right? so why not add isChrome check?

Reply:
@makarandp0 No, the fix is to disable RTX in Firefox 79+ SDPs. So, the fix is on the Firefox side, if you notice carefully.

Reply:
ofcourse! its only for firefox :) I misread `!isFirefox` I think it would make it easier to switch this to
```
if (isFirefox) {
   // disable RTX for Firefox SDPs.
   offer = workaround 1106157(offer);
} else {
  offer = workaroundIssue8329(offer);
}
```